### ISSUE TITLE
MD049 collides with prettier

### ISSUE THREAD

**Issue by viceice** [DavidAnson/markdownlint#483]
https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md049---emphasis-style-should-be-consistent

That rule collides with prettier here

https://github.com/renovatebot/renovate/blob/11f4e1e8564b2f4f9e4a53753166a2a4d93438a9/docs/usage/getting-started/private-packages.md?plain=1#L126

```md
If a `github.com` token is found in `hostRules`, then it is written out to local [GIT*CONFIG*](https://git-scm.com/docs/git-config#Documentation/git-config.txt-GITCONFIGCOUNT) variables prior to running `go` commands.
```

ref:
- https://github.com/renovatebot/renovate/pull/13613
- https://github.com/DavidAnson/markdownlint/pull/448

**Comment by viceice** [comment]
Workaround is to use `<!-- markdownlint-disable-next-line MD049 -->` for that line

**Comment by DavidAnson** [comment]
What does "collides with Prettier" mean? The line you show does not produce any errors on its own:

https://dlaa.me/markdownlint/#%25m%23%20Issue%20483%0A%0AIf%20a%20%60github.com%60%20token%20is%20found%20in%20%60hostRules%60%2C%20then%20it%20is%20written%20out%20to%20local%20%5BGIT*CONFIG*%5D(https%3A%2F%2Fgit-scm.com%2Fdocs%2Fgit-config%23Documentation%2Fgit-config.txt-GITCONFIGCOUNT)%20variables%20prior%20to%20running%20%60go%60%20commands.%0A

It looks like it contains an asterisk emphasis, so if MD049 as already seen an underscore emphasis, it will report this line for being inconsistent (correctly).

What is the issue with Prettier?

**Comment by viceice** [comment]
Prettier will do the opposite. So it will flip flop between star and underscore.

**Comment by DavidAnson** [comment]
Does Prettier have a policy to only use a certain character for emphasis? If so, this rule can be configured to use that same character. Or, if that character is always preferred, this rule will naturally allow that because the default behavior looks for consistency within the file.

Can you link to the relevant documentation for Prettier, please?

**Comment by nschonni** [comment]
`GIT*CONFIG*`: GIT*CONFIG*
vs.
`GIT_CONFIG_`: GIT_CONFIG_

looks like a functional issue. It seems like the latter is the correct format if the intention was to say something like `GIT_CONFIG_*`

**Comment by nschonni** [comment]
FYI, under the hood, prettier uses Remark, so it's defaults come from that library

**Comment by viceice** [comment]
Ok, I'll check again what we like to achieve.
